Convert bucket aggs docs to runtime fields (#71202)
elastic/elasticsearch: docs/reference/aggregations/bucket/composite-aggregation.asciidoc

}
--------------------------------------------------

> Like the `terms` aggregation it is also possible to use a
> <<runtime,runtime field>> to create the values for the composite buckets:

Reviewer: ```suggestion
Like the `terms` aggregation, it's possible to use a
<<runtime,runtime field>> to create values for the composite buckets:
```

}
--------------------------------------------------

> Like the `histogram` aggregation it is also possible to use a
> <<runtime,runtime field>> to create the values for the composite buckets:

Reviewer: ```suggestion
Like the `histogram` aggregation it's possible to use a
<<runtime,runtime field>> to create values for the composite buckets:
```
